edx_sga/static/js/src/edx_sga.js

```diff
@@ -242,7 +242,7 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                 var max_score = row.parents('#grade-info').data('max_score');
                 var score = Number(form.find('#grade-input').val());
                 event.preventDefault();
-                if (!score) {
+                if (!score && parseInt(score) !== 0) {
```

Comment:
`Number` should already convert `score` to a number or `NaN`

Reply:
Can u ellaborate a bit more ? 
Actually, in JS  !0 evaluate to true